frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -458,7 +464,7 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
               isValid={!isValidationError(busValidation)}
               value={asFormSelectValue(bus)}
               id={asId('interface')}
-              isDisabled={isDisabled('interface')}
+              isDisabled={isDisabled('interface') || isWindowsCD}
```

Comment:
IMO we should still allow the user to change it. Anyway generally, it is better to pass these values in editConfig instead from outside

Reply:
Why give the illusion that the user can change the interface?
the only wanted bus for windows && cd is SATA so it should be fixed to SATA.
thats why this change shouldn't be deprecated, IMO

Reply:
what about scsi? IMo we should use only what comes from the template and don't have a special logic like this one